app/src/ui/create-tag/create-tag-dialog.tsx

```diff
@@ -43,12 +45,14 @@ export class CreateTag extends React.Component<
     this.state = {
       tagName: props.initialName || '',
       isCreatingTag: false,
+      previousTags: this.getExistingTagsFiltered(),
     }
   }
 
   public render() {
     const error = this.getCurrentError()
     const disabled = error !== null || this.state.tagName.length === 0
+    const lastThreeTags = _.takeRight(this.state.previousTags, 3)
```

Comment:
```suggestion
    const lastThreeTags = this.state.previousTags.slice(0, 3)
```
LoDash isn't needed for this operation from what I can tell

Reply:
But `slice` will take the elements from the left, no? `_.takeRight` will return the 3 elements at the end, while `.slice(0,3)` will return the first 3 elements 🤔

Reply:
I agree but a couple things
1. Why do we want the last 3 items to begin with? Is this because they are the oldest? Should we sort by something like creation time?
2. We can always use `.slice(-3)` to get the last 3 items

Reply:
1. They are supposed to be the last 3 (the newest)
2. Good point!

I think I'll merge this tomorrow, address the `.slice` thing and hide this behind a feature flag 😄  The team agreed today that this should be ready to go 🚀